### ISSUE TITLE
[BUG][IOS] Can't write after coming back from background due to access control checks

### ISSUE THREAD

**Issue by dragermrb** [diachedelic/capacitor-blob-writer#53]
## Bug Report

On IOS and capacitor4, can't write after coming back from background due to access control checks. This is the error reported on console:

```
[Error] Could not connect to the server.
[Error] Fetch API cannot load http://localhost:53869/var/mobile/Containers/Data/Application/F7A3241E-E154-4C7C-A360-9971CD09A1E3/Documents/demo_0.10861703844695692.pdf due to access control checks.
[Error] Failed to load resource: Could not connect to the server. (demo_0.10861703844695692.pdf, line 0)
```

### Versions
- @capacitor/cli: 4.3.0
- @capacitor/core: 4.3.0
- @capacitor/ios: 4.3.0
- @capacitor/filesystem@4.1.1
- capacitor-blob-writer@1.1.3

### Current Behavior

After putting the app in the background and turning off the screen for a few seconds, it is not possible to write files again.

Follow the steps below to reproduce the issue:

- Open app on IOS (real device)
- Download some file and save it (`write_blob(...)`)
- Put app on background
- Turn off the device screen for 5 seconds
- Turn on screen a return to app
- Try to download another file and save it (`write_blob(...)`).
- Nothing is saved and can see the following errors on safari web console:

```
[Error] Could not connect to the server.
[Error] Fetch API cannot load http://localhost:53869/var/mobile/Containers/Data/Application/F7A3241E-E154-4C7C-A360-9971CD09A1E3/Documents/demo_0.10861703844695692.pdf due to access control checks.
[Error] Failed to load resource: Could not connect to the server. (demo_0.10861703844695692.pdf, line 0)
```

### Code Reproduction
Sample app to reproduce de issue

https://github.com/dragermrb/app-sample-blob-writer

```sh
git clone git@github.com:dragermrb/app-sample-blob-writer.git
cd app-sample-blob-writer
npm install
npm run build
npx cap sync ios
```

**Comment by diachedelic** [comment]
What happens if you change this line to `true`? https://github.com/diachedelic/capacitor-blob-writer/blob/cb517d6c82920f90226ec07b6ddf6dfebc5c8f9d/ios/Plugin/Plugin.swift#L77

**Comment by dragermrb** [comment]
It's working!

**Comment by diachedelic** [comment]
I set that option to false to fix #49. What version of iOS are you using?

**Comment by diachedelic** [comment]
This should be fixed in v1.1.4, please test.

**Comment by dragermrb** [comment]
> This should be fixed in v1.1.4, please test.

I am using iOS version 15.6.1. 

Now it works fine with version v1.1.4.

Thanks!!
